openpype/hosts/maya/api/lib.py

```diff
@@ -2091,44 +2100,25 @@ def get_frame_range():
     if handle_end is None:
         handle_end = handles
 
-    return {
-        "frameStart": frame_start,
-        "frameEnd": frame_end,
-        "handleStart": handle_start,
-        "handleEnd": handle_end
-    }
-
-
-def reset_frame_range(playback=True, render=True, fps=True):
```

Comment:
This is still used on two places - without it Maya integration won't even start and reset frame range menu won't work.

**lib_rendersettings.py**
```
17: from openpype.hosts.maya.api.lib import reset_frame_range
161 reset_frame_range(playback=False, fps=False, render=True)
```

**menu.py**
```
command=lambda *args: lib.reset_frame_range()
```

Reply:
Thank you so much! Totally missed the fact that it somehow removed the function due to how it showed the changes on Github. Agreed, this should definitely be reverted.